tests/composition_names_duplicates/screen-output

```diff
@@ -15,7 +15,7 @@ Number of degrees of freedom: 49 (18+4+9+9+9)
 An exception happened on MPI process <0> while running the visualization postprocessor <N6aspect11Postprocess27VisualizationPostprocessors18MaterialPropertiesILi2EEE>: 
 
 An error occurred in file <visualization.cc> in function
-(line in output replaced by default.sh script)
+    void aspect::Postprocess::{anonymous}::add_data_names_to_set(const std::vector<std::__cxx11::basic_string<char> >&, std::set<std::__cxx11::basic_string<char> >&)
```

Comment:
the problem is that the arguments are formatted slightly differently if you run with a different compiler (for example ``__cxx11::basic_string here``).

Reply:
Oh darn. Ok, I've re-removed those lines and squashed the last two commits.